index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>How to write well-formed XHTML</title>
+  <meta charset="utf-8">
```

Comment:
This is where I'd hope to see you link out to external scripts - rather than down in your body.